### ISSUE TITLE
Custom template for output

### ISSUE THREAD

**Issue by Khrystyna** [ichord/At.js#515]
Hi! Great plugin! I want to integrate it into my system, so I'd love to support own custom template for mentioning. Library generates `<span class="atwho-inserted" data-atwho-at-query="@">Simon</span>`, which is different from what I have now. Can I change this template with my own? Thanks in advance!

**Comment by markdon** [comment]
Yeah, it's insertTpl. Check out the [settings here](https://github.com/ichord/At.js/wiki/Base-Document#base-usage).

**Comment by farlock85** [comment]
@markdon old comment, but not what @Khrystyna was on about.

There doesn't seem to be a way to format the output HTML tags with custom attributes (like inserting a userID).
